### PR TITLE
chore: Update platform and component descriptions

### PR DESCRIPTION
**Pull request by binarylogic**
This change aims to update all of our platform and component descriptions. I've rewritten these by hand, but was inspired by their previous descriptions.

- [X] Platform descriptions
- [ ] Source descriptions
- [ ] Sink descriptions

### CODE REVIEW

File: .meta/_partials/descriptions/_aws_cloudwatch.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 description = """\
-[Amazon CloudWatch][urls.aws_cloudwatch] is a monitoring and management \
-service that provides data and actionable insights for AWS, hybrid, and \
-on-premises applications and infrastructure resources. With CloudWatch, you \
-can collect and access all your performance and operational data in the form \
-of logs and metrics from a single platform.\
+[Amazon CloudWatch][urls.aws_cloudwatch] is Amazon Web Service's (AWS) \
+embedded monitoring service designed to monitor AWS resources. It \
+automatically integration with AWS infrastructure to help you monitor \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_aws_cloudwatch.toml:4]: integration should be integrates.

File: .meta/_partials/descriptions/_aws_cloudwatch.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+cloud infrastructure, services, and applications thourgh logs, metrics, \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_aws_cloudwatch.toml:5]: through.

File: .meta/_partials/descriptions/_aws_cloudwatch.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+traces, and other operatinal data.\
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_aws_cloudwatch.toml:6]: operational.

File: .meta/_partials/descriptions/_datadog.toml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 description = """\
-[Datadog][urls.datadog] is a monitoring service for cloud-scale applications, \
-providing monitoring of servers, databases, tools, and services, through a \
-SaaS-based data analytics platform.\
+[Datadog][urls.datadog] is a managed monitoring service for cloud-based \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_datadog.toml:2]: Cloud

File: .meta/_partials/descriptions/_datadog.toml [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
+applications. It allows you to monior servers, databases, and other services \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_datadog.toml:3]: monitor

File: .meta/_partials/descriptions/_docker.toml [hunk 6]
```diff
@@ -1,9 +1,7 @@
...
-[Docker][urls.docker] is an open platform for developing, shipping, and running
-applications and services. Docker enables you to separate your services from \
-your infrastructure so you can ship quickly. With Docker, you can manage your \
-infrastructure in the same ways you manage your services. By taking advantage \
-of Docker’s methodologies for shipping, testing, and deploying code quickly, \
-you can significantly reduce the delay between writing code and running it in \
-production.\
+[Docker][urls.docker] is a container platform for defining environments that \
+run applications and services. Docker separates your environments from your \
+infrastructure, making your application and services portable. This reduces \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_docker.toml:4]: applications

File: .meta/_partials/descriptions/_gcp_stackdriver.toml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 description = """\
-[Stackdriver][urls.gcp_stackdriver] is Google Cloud's embedded observability \
-suite designed to monitor, troubleshoot, and improve cloud infrastructure, \
-software and application performance. Stackdriver enables you to efficiently \
-build and run workloads, keeping applications available and performing well.\
+[Stackdriver][urls.gcp_stackdriver], also know as Operations, is Google \
+Cloud's embedded monitoring suite designed to monitor cloud infrastructure, \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_gcp_stackdriver.toml:3]: Cloud

File: .meta/_partials/descriptions/_kafka.toml [hunk 8]
```diff
@@ -1,8 +1,8 @@
 description = """\
-[Apache Kafka][urls.kafka] is an open-source project for a distributed \
-publish-subscribe messaging system rethought as a distributed commit log. \
-Kafka stores messages in topics that are partitioned and replicated across \
-multiple brokers in a cluster. Producers send messages to topics from which \
-consumers read. These features make it an excellent candidate for durably \
-storing logs and metrics data.\
+[Apache Kafka][urls.kafka] is an open-source project for streaming data. It \
+a distributed commit log, similar to publish-subscribe messaging systems. \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_kafka.toml:3]: Is is

File: .meta/_partials/descriptions/_kafka.toml [hunk 9]
```diff
@@ -1,8 +1,8 @@
...
+Messages are streamed into Kafka topics and split across 1 or more partitions. \
+Data is replicated across multiple brokers ensuring high durability. This \
+makes it an excellent candidate for durabily streaming logs, metrics, traces, \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_kafka.toml:6]: durably

File: .meta/_partials/descriptions/_kubernetes.toml [hunk 10]
```diff
@@ -1,9 +1,9 @@
...
-applications across a cluster of nodes. In simple terms, you have a group of \
-machines (e.g. VMs) and containerized applications (e.g. Dockerized \
-applications), and Kubernetes will help you to easily manage those apps across \
-those machines. Because Kubernetes is very abstract it is critically important \
-that you collect the appropriate logs and metrics to monitor and observe \
-Kubernetes activity.\
+[Kubernetes][urls.kubernetes] is a production-grade container orchestration \
+platform. It allows you to manage containerized (Dockerized) applications and \
+services across a cluster of nodes. It automates resource management, helping \
+you to manage and deploy applicationd and services across a variable number \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_kubernetes.toml:5]: applications

File: .meta/_partials/descriptions/_prometheus.toml [hunk 11]
```diff
@@ -1,6 +1,8 @@
 description = """\
-[Prometheus][urls.prometheus] is a pull-based monitoring system that scrapes \
-metrics from configured endpoints, stores them efficiently, and supports a \
-powerful query language to compose dynamic information from a variety of \
-otherwise unrelated data points.\
+[Prometheus][urls.prometheus] is an open-source metrics database for \
+monitoring infrastructure, services, and applications. It's built upon a \
+unique pull-based model where Prometheus scrapes upstream service endpoints \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_prometheus.toml:4]: -unique

File: .meta/_partials/descriptions/_prometheus.toml [hunk 12]
```diff
@@ -1,6 +1,8 @@
...
+that expose internal metrics data. This model increases Prometheus' \
+reliability, making it well suited for cloud-based environments where metrics \
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_prometheus.toml:6]: Cloud-based

File: .meta/_partials/descriptions/_statsd.toml [hunk 13]
```diff
@@ -1,5 +1,6 @@
 description = """\
-[StatsD][urls.statsd] is a standard and, by extension, a set of tools that can \
-be used to send, collect, and aggregate custom metrics from any application. \
-Originally, StatsD referred to a daemon written by [Etsy][urls.etsy] in Node.\
+[StatsD][urls.statsd], created by [Etsy][urls.etsy], is a network daemon that \
+listens for metrics data from upstream services, applications, and agents. \
+Since its inception, StatsD has grown into a standard for defining metrics \
+data where it is widely supported by a variety of metrics solutions.\
```
Comment (jamtur01) [inline .meta/_partials/descriptions/_statsd.toml:5]: where -> and

**Review by jamtur01 — CHANGES_REQUESTED** [review]
Some spelling and grammar.

File: .meta/_partials/descriptions/_datadog.toml [hunk 14]
```diff
@@ -1,5 +1,5 @@
...
+through a SaaS-based monitoring platform.\
```
Comment (jszwedko) [inline .meta/_partials/descriptions/_datadog.toml:4]: I'd probably avoid abbreviations and do `Software-as-a-Service` if it isn't previously defined and abbreviated in the content.

**Review by jszwedko — COMMENTED** [review]
Thanks for running through these!

**Comment by jamtur01** [comment]
@binarylogic Should this be closed in light of cue updates?

**Comment by binarylogic** [comment]
Yep

**Pull request closed without merging**